src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
+          description: Start of the current billing period for the Account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/History'
+                $ref: '#/components/schemas/BillingDate'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/notify_settings':
+  '/usage/vectors':
     get:
-      operationId: GetBillingNotifySettings
+      operationId: GetUsageVectors
       tags:
-        - BillingNotifySettings
+        - Usage
       responses:
         '200':
-          description: Settings for billing notifications
+          description: Usage vectors for the account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingNotifySettings'
+                $ref: '#/components/schemas/UsageVectors'
+
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-    put:
-      operationId: PutBillingContact
+  '/usage/{type}':
+    get:
+      operationId: GetUsagetype
       tags:
-        - PaymentMethods
-      requestBody:
-        description: Billing Contact to Update
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/BillingNotifySettings'
+        - Usage
+      parameters:
+        - in: path
+          name: type
+          required: true
+          schema:
+            type: string
+            enum: [limits, billing, storage_gb, write_mb, reads_gb, query_count]
+          description: The usage type or vector to query. Can be a usage metric, billing stats, or rate limit events
+          example: 'reads_gb'
+        - in: query
+          name: start
+          schema:
+            type: string
+            enum: [24h, 7d, 30d]
+            default: 24h
       responses:
         '200':
-          description: billing notification settings updated
+          description: Usage history for account
           content:
-            application/json:
+            application/csv:
               schema:
-                $ref: '#/components/schemas/BillingNotifySettings'
+                type: string
+                example: |
+                  #group,false,false,true,false,false
+                  #datatype,string,long,string,dateTime:RFC3339,long
+                  #default,reads_gb,,,,
+                  ,result,table,_field,_time,_value
+                  ,,0,reads_gb,2021-01-13T22:05:00Z,120
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/contact':
+  '/settings/notifations':
```

Comment:
```suggestion
  '/settings/notifications':
```

Reply:
Is this distinct enough from a `Billing` to warrant its own object/route? Wonder if it makes sense to roll this under that section.

Reply:
Because there is a `put` route for it as well, it makes sense to keep it separated.